frb_example/pure_dart/rust/.frb.yml

```diff
@@ -0,0 +1,10 @@
+rust_input: [src/api.rs]
```

Comment:
nit: Given this frb.yml, maybe we should modify frb_codegen/src/main.rs's unit tests (introduced at https://github.com/fzyzcjy/flutter_rust_bridge/pull/1106) to use this config, thus remove duplication

Reply:
Running into issues with changing working directories in one test messing up another. Perhaps a `--config-file` options is in order?

Reply:
> Running into issues with changing working directories in one test messing up another.

Hmm not sure the exact issue. One way may be, shall we let frb think the base directory is the dir with .frb.yml?

Reply:
Better idea, since we aren't using positional arguments for the CLI, we could have it accept a single config file or a directory containing one, and we add the same entry point for the library as well.